README.md

```diff
@@ -33,6 +33,8 @@ Kokkos::fence();
 KokkosFFT::rfft(execution_space(), x, x_hat);
 ```
 
+There are two major differences: ```execution_space``` argument and updating output as argument not returned. As imagined, KokkosFFT only accepts [Kokkos Views](https://kokkos.org/kokkos-core-wiki/API/core/View.html) as input data. The accessibilities of Views from ```execution_space``` is statically checked (compilation errors if not accessible). 
```

Comment:
> updating output as argument not returned

Not very clear

Also try to define/link to `execution_space`.

Reply:
I will add a link to `execution_space`. How about rephrasing as 
"_output value is an argument of APIs (not returned from APIs)._"